docs/reference/repositories-metering-api/apis/repositories-meterings-body.asciidoc

```diff
@@ -0,0 +1,135 @@
+tag::repositories-metering-body[]
+`<node_id>`::
+(Optional, string) Comma-separated list of node IDs or names used to limit
+returned information.
+`_nodes`::
+(object)
+Contains statistics about the number of nodes selected by the request.
++
+.Properties of `_nodes`
+[%collapsible%open]
+====
+`total`::
+(integer)
+Total number of nodes selected by the request.
+
+`successful`::
+(integer)
+Number of nodes that responded successfully to the request.
+
+`failed`::
+(integer)
+Number of nodes that rejected the request or failed to respond. If this value
+is not `0`, a reason for the rejection or failure is included in the response.
+====
+
+`cluster_name`::
+(string)
+Name of the cluster. Based on the <<cluster.name>> setting.
+
+`nodes`::
+(object)
+Contains repositories metrics for the nodes selected by the request.
++
+.Properties of `nodes`
+[%collapsible%open]
+====
+`<node_id>`::
+(array)
+An array of repository metrics for the node.
++
+.Properties of objects in `node_id`
+[%collapsible%open]
+=====
+`repository_name`::
+(string) 
+Repository name.
+
+`repository_type`::
+(string) 
+Repository type.
+
+`repository_location`::
+(string) 
+Represents an unique location within the repository, 
+i.e. a `bucket` + `base_path`.
+
+`repository_ephemeral_id`::
+(string) 
+An identifier that changes every time the repository is updated.
+
+`repository_started_at`::
+(long)
+Time the repository was created or updated. Recorded in milliseconds
+since the https://en.wikipedia.org/wiki/Unix_time[Unix Epoch].
+
+`repository_stopped_at`::
+(long)
+Time the repository was deleted or updated. Recorded in milliseconds
+since the https://en.wikipedia.org/wiki/Unix_time[Unix Epoch].
+
+`archived`::
+(boolean) 
+A flag that tells whether or not this object has been archived.
+
+`cluster_version`:: 
```

Comment:
I can see the confusion with this version field, and how to use it correctly. This needs  documentation (i.e. with more detail on how to use it), also mentioning caveats etc.